sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -490,7 +490,7 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
     }
   }
 
-  test("Fiters should be pushed down for vectorized Parquet reader at row group level") {
+  ignore("Fiters should be pushed down for vectorized Parquet reader at row group level") {
```

Comment:
Is it possible we can assign an custom id to an accumulator? So we can retrieve it back in executor side with the custom id.